Core.ApplicationServices/AdviceService.cs

```diff
@@ -93,16 +93,18 @@ public IQueryable<Advice> GetAdvicesForOrg(int orgKey)
             return result.AsQueryable();
         }
 
-        public IQueryable<Advice> GetAdvicesFromCurrentUsersOrganizationMemberships()
+        public IQueryable<Advice> GetAccessibleCurrentToUser()
         {
-            return OrganizationalUserContext
-                .OrganizationIds
-                .Select(GetAdvicesForOrg)
-                .Aggregate<IQueryable<Advice>, IQueryable<Advice>>
-                (
-                    null,
-                    (acc, next) => acc == null ? next : acc.Concat(next)
-                );
+            return OrganizationalUserContext.IsGlobalAdmin()
+                ? AdviceRepository.AsQueryable()
```

Comment:
Her tager den bare hele advice repository og siger god for? Så må der vel være noget sortering længere ude?

Reply:
Ja det her er "root" query. Det er alt hvad brugern må få og det sendes videre til filtrering via odata (se controlleren). Det er også sådan de fleste andre fungerer men fordi advis er så sær, så skal vi skrive lidt ekstra kode